src/parser/parseReactElement.js

```diff
@@ -1,15 +1,16 @@
 /* @flow */
 
-import * as React from 'react';
-import type { Options } from './../options';
+import React from 'react';
 import {
   createStringTreeNode,
   createNumberTreeNode,
   createReactElementTreeNode,
 } from './../tree';
+import type { Element as ReactElement } from 'react';
```

Comment:
You could use one import from react
import React, { type Element as ReactElement } from 'react';

Reply:
Yep, that's work. Awesome! Thanks a lot!

Reply:
`import React, { type Element as ReactElement } from 'react';`

Ho that's a good one! I didn't now it :D